AzureCalling/Controllers/CallViewController.swift

```diff
@@ -731,7 +731,14 @@ class CallViewController: UIViewController, UICollectionViewDelegate, UICollecti
 
     private func promptForTeamsAdmissionDenied() {
         presentAlert(title: "Sorry, you were denied access to the meeting") { [weak self] in
-            self?.navigationController?.popViewController(animated: true)
+            guard let self = self else {
+                return
+            }
+
+            self.navigationController?.popViewController(animated: true)
+            if let lobbyViewController = self.navigationController?.topViewController as? LobbyViewController {
```

Comment:
Could this function be moved into LobbyViewController's viewController lifecycle method: ViewWillAppear?

Reply:
Good point!